### ISSUE TITLE
Is it possible to register my service in more than one Eureka at the same time?

### ISSUE THREAD

**Issue by torrespro** [spring-cloud/spring-cloud-netflix#2983]
In our architecture some presentation microservices and some integration microservices providing data. We have multiple testing environments and we want to reuse the same set of integration microservices between test1, test2 and test3 environments to avoid deploying all of them 3 times one per environments.

Will be possible to have the same presentation services, gateway and registry 3 times but share only 1 set of integration microservices registering them into the 3 eurekas?

Not talking about:

`eureka.client.serviceUrl.defaultZone=http://<peer1host>:<peer1port>/eureka,http://<peer2host>:<peer2port>/eureka`

Because then they only show up in the first registry. Something to have them in the 3 registries at the same time and attending calls from the 3 gateway at the same time.

**Comment by spencergibb** [comment]
No. It should show up in the other peers if the eureka servers are setup to peer to each other.

**Comment by torrespro** [comment]
If I setup them to peer each other, the service will show up in all the eurekas? In my tested service went to the first Eureka and only shows up in the other ones if I stop the Eureka1

**Comment by spencergibb** [comment]
It should if peering is setup correctly. That is what peering is for.

**Comment by spencergibb** [comment]
you should close the stack overflow issue as well. We monitor both.
